### YamlMime:ManagedReference
items:
- uid: System.Xaml.XamlException
  id: XamlException
  children:
  - System.Xaml.XamlException.#ctor
  - System.Xaml.XamlException.#ctor(System.String)
  - System.Xaml.XamlException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Xaml.XamlException.#ctor(System.String,System.Exception)
  - System.Xaml.XamlException.#ctor(System.String,System.Exception,System.Int32,System.Int32)
  - System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Xaml.XamlException.LineNumber
  - System.Xaml.XamlException.LinePosition
  - System.Xaml.XamlException.Message
  langs:
  - csharp
  name: XamlException
  nameWithType: XamlException
  fullName: System.Xaml.XamlException
  type: Class
  summary: "A kivétellel, hogy az általános XAML-olvasó vagy XAML-író kivételt vált ki. Tekintse meg a megjegyzés."
  remarks: "Például több egyedi kivételek <xref:System.Xaml.XamlParseException>létezik, és gyakran okozott helyett XamlException.</xref:System.Xaml.XamlParseException> Bizonyos kivételek kapcsán általában pontosabb kivételek adatai tulajdonságként a kivétel, vagy egyszerűen adja meg, mert az adott kivételt észlelt."
  syntax:
    content: 'public class XamlException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Xaml.XamlException.#ctor
  id: '#ctor'
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: XamlException()
  nameWithType: XamlException.XamlException()
  fullName: System.Xaml.XamlException.XamlException()
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlException&quot;> </xref> osztály. A példány egy rendszer által megadott üzenetet, amely leírja a hibát tartalmaz."
  syntax:
    content: public XamlException ();
    parameters: []
  overload: System.Xaml.XamlException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: XamlException(String)
  nameWithType: XamlException.XamlException(String)
  fullName: System.Xaml.XamlException.XamlException(String)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlException&quot;> </xref> osztály. A példány a megadott üzenet, amely leírja a hibát tartalmaz."
  syntax:
    content: public XamlException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe kell győződjön meg arról, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
  overload: System.Xaml.XamlException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: XamlException(SerializationInfo,StreamingContext)
  nameWithType: XamlException.XamlException(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlException.XamlException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlException&quot;> </xref> a szerializált adatok osztályt."
  syntax:
    content: protected XamlException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A szerializált objektum adatokat tartalmazó objektum."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és környezeti információt."
  overload: System.Xaml.XamlException.#ctor*
  exceptions:
  - type: System.ArgumentNullException
    commentId: T:System.ArgumentNullException
    description: "<code>info</code>is <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>."
  platform:
  - net462
- uid: System.Xaml.XamlException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: XamlException(String,Exception)
  nameWithType: XamlException.XamlException(String,Exception)
  fullName: System.Xaml.XamlException.XamlException(String,Exception)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlException&quot;> </xref> osztály. A példány megadott hibaüzenet és a belső kivételben találhatók okozza-e ezt a kivételt, egy hivatkozást tartalmaz."
  remarks: "Kivételt vált ki, mert egy előző kivétel az előző kivétel hivatkozni kell a <xref:System.Exception.InnerException%2A>tulajdonság.</xref:System.Exception.InnerException%2A> A <xref:System.Exception.InnerException%2A>tulajdonság; a konstruktornak átadott ugyanazt az értéket adja vissza, vagy adja vissza, `null`Ha a <xref:System.Exception.InnerException%2A>tulajdonság nem ad meg a belső kivétel értéket a konstruktor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XamlException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe kell győződjön meg arról, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza. Ha a `innerException` paraméter nem <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a jelenlegi kivétel a következik be egy <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blokk, amely kezeli a belső kivételben találhatók."
  overload: System.Xaml.XamlException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlException.#ctor(System.String,System.Exception,System.Int32,System.Int32)
  id: '#ctor(System.String,System.Exception,System.Int32,System.Int32)'
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: XamlException(String,Exception,Int32,Int32)
  nameWithType: XamlException.XamlException(String,Exception,Int32,Int32)
  fullName: System.Xaml.XamlException.XamlException(String,Exception,Int32,Int32)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlException&quot;> </xref> osztály. A példány a megadott hibaüzenet, a belső kivételben és sor információkat tartalmaz."
  syntax:
    content: public XamlException (string message, Exception innerException, int lineNumber, int linePosition);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe kell győződjön meg arról, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza."
    - id: lineNumber
      type: System.Int32
      description: "A sor számát a hibakeresés vagy egy sor információk fogyasztók."
    - id: linePosition
      type: System.Int32
      description: "A pozíció hibakeresés vagy sor információk fogyasztók jelentheti."
  overload: System.Xaml.XamlException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: XamlException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlException.GetObjectData(SerializationInfo,StreamingContext)
  type: Method
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Megvalósítja <xref:System.Runtime.Serialization.ISerializable.GetObjectData*>és szerializálási támogatást nyújt a sor adatok.</xref:System.Runtime.Serialization.ISerializable.GetObjectData*>"
  syntax:
    content: public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A szerializált objektum adatokat tartalmazó objektum."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és környezeti információt."
  overload: System.Xaml.XamlException.GetObjectData*
  exceptions:
  - type: System.ArgumentNullException
    commentId: T:System.ArgumentNullException
    description: "<code>info</code>is <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>."
  platform:
  - net462
- uid: System.Xaml.XamlException.LineNumber
  id: LineNumber
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: LineNumber
  nameWithType: XamlException.LineNumber
  fullName: System.Xaml.XamlException.LineNumber
  type: Property
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Lekérdezi vagy beállítja a vonal összetevőt XAML szöveg sor információ, amely a kivételt jelent."
  syntax:
    content: public int LineNumber { get; protected set; }
    return:
      type: System.Int32
      description: "A sor összetevőt az XAML szöveg sor információ."
  overload: System.Xaml.XamlException.LineNumber*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlException.LinePosition
  id: LinePosition
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: LinePosition
  nameWithType: XamlException.LinePosition
  fullName: System.Xaml.XamlException.LinePosition
  type: Property
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Lekérdezi vagy beállítja a sor pozícióját összetevője XAML szöveg sor információk a kivételi jelentést."
  syntax:
    content: public int LinePosition { get; protected set; }
    return:
      type: System.Int32
      description: "A sor pozícióját összetevője XAML szöveginformációkat a sor."
  overload: System.Xaml.XamlException.LinePosition*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlException.Message
  id: Message
  parent: System.Xaml.XamlException
  langs:
  - csharp
  name: Message
  nameWithType: XamlException.Message
  fullName: System.Xaml.XamlException.Message
  type: Property
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Lekérdezi vagy beállítja a kivételre vonatkozó üzenet, és adatai nem érhető el, ha fűzi hozzá a sor információkat az üzenetet."
  remarks: "<xref:System.Xaml.XamlException>Felülírja ezt a tulajdonságot, hogy a sor adatokat közvetlenül az üzenetben szereplő, ezért könnyebben látható hibakereső vagy más kivétel felhasználói.</xref:System.Xaml.XamlException> Sor információ nem érhető el, ha a ezt a tulajdonságot a sor információkat tartalmazza. Ha sor információ nem érhető el, a következő: ugyanaz, mint az alap üzenet."
  syntax:
    content: public override string Message { get; }
    return:
      type: System.String
      description: "A kivételre vonatkozó üzenet, amely a hozzáfűzött adatait tartalmazza."
  overload: System.Xaml.XamlException.Message*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.ArgumentNullException
  isExternal: true
  name: System.ArgumentNullException
- uid: System.Xaml.XamlException.#ctor
  parent: System.Xaml.XamlException
  isExternal: false
  name: XamlException()
  nameWithType: XamlException.XamlException()
  fullName: System.Xaml.XamlException.XamlException()
- uid: System.Xaml.XamlException.#ctor(System.String)
  parent: System.Xaml.XamlException
  isExternal: false
  name: XamlException(String)
  nameWithType: XamlException.XamlException(String)
  fullName: System.Xaml.XamlException.XamlException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Xaml.XamlException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Xaml.XamlException
  isExternal: false
  name: XamlException(SerializationInfo,StreamingContext)
  nameWithType: XamlException.XamlException(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlException.XamlException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Xaml.XamlException.#ctor(System.String,System.Exception)
  parent: System.Xaml.XamlException
  isExternal: false
  name: XamlException(String,Exception)
  nameWithType: XamlException.XamlException(String,Exception)
  fullName: System.Xaml.XamlException.XamlException(String,Exception)
- uid: System.Xaml.XamlException.#ctor(System.String,System.Exception,System.Int32,System.Int32)
  parent: System.Xaml.XamlException
  isExternal: false
  name: XamlException(String,Exception,Int32,Int32)
  nameWithType: XamlException.XamlException(String,Exception,Int32,Int32)
  fullName: System.Xaml.XamlException.XamlException(String,Exception,Int32,Int32)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Xaml.XamlException
  isExternal: false
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: XamlException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlException.GetObjectData(SerializationInfo,StreamingContext)
- uid: System.Xaml.XamlException.LineNumber
  parent: System.Xaml.XamlException
  isExternal: false
  name: LineNumber
  nameWithType: XamlException.LineNumber
  fullName: System.Xaml.XamlException.LineNumber
- uid: System.Xaml.XamlException.LinePosition
  parent: System.Xaml.XamlException
  isExternal: false
  name: LinePosition
  nameWithType: XamlException.LinePosition
  fullName: System.Xaml.XamlException.LinePosition
- uid: System.Xaml.XamlException.Message
  parent: System.Xaml.XamlException
  isExternal: false
  name: Message
  nameWithType: XamlException.Message
  fullName: System.Xaml.XamlException.Message
- uid: System.Xaml.XamlException.#ctor*
  parent: System.Xaml.XamlException
  isExternal: false
  name: XamlException
  nameWithType: XamlException.XamlException
- uid: System.Xaml.XamlException.GetObjectData*
  parent: System.Xaml.XamlException
  isExternal: false
  name: GetObjectData
  nameWithType: XamlException.GetObjectData
- uid: System.Xaml.XamlException.LineNumber*
  parent: System.Xaml.XamlException
  isExternal: false
  name: LineNumber
  nameWithType: XamlException.LineNumber
- uid: System.Xaml.XamlException.LinePosition*
  parent: System.Xaml.XamlException
  isExternal: false
  name: LinePosition
  nameWithType: XamlException.LinePosition
- uid: System.Xaml.XamlException.Message*
  parent: System.Xaml.XamlException
  isExternal: false
  name: Message
  nameWithType: XamlException.Message
